lib/platform/linux-i2c.c

```diff
@@ -48,10 +49,44 @@ struct switchtec_i2c {
 	int i2c_addr;
 };
 
+#define CMD_GET_CAP  0xE0
+#define CMD_GET_WRITE_STATUS  0xE2
+#define CMD_GAS_WRITE_WITH_STATUS  0xE8
+#define CMD_GAS_READ  0xE9
+#define CMD_GAS_WRITE  0xEA
+
+#define MAX_RETRY_COUNT  100
+
+#define PEC_BYTE_COUNT  1
+#define DATA_TAIL_BYTE_COUNT  2
+
+#define CAP_MASK  0x80
```

Comment:
This seems like a very poor name. What is the 8th bit actually mean? I don't think CAP_MASK describes it.